koku/masu/database/report_manifest_db_accessor.py

```diff
@@ -309,6 +309,20 @@ def update_and_get_day_file(self, day, manifest_id):
             manifest.save(update_fields=["report_tracker"])
             return f"{day}_{counter}.csv"
 
+    def update_and_get_parquet_batch_counter(self, day, manifest_id):
+        """This is needed for OCP on Cloud filtered daily parquet files"""
+        with transaction.atomic():
```

Comment:
What happens on a collision? Does one worker wait then retry? I'm not familiar with exactly how these locks work.

Reply:
Correct this is the same logic our counters use for csv files already